lib/workers/repository/updates/branch-name.ts

```diff
@@ -64,6 +64,10 @@ export function generateBranchName(update: RenovateConfig): void {
         update.groupSlug = `major-${update.groupSlug}`;
       }
     }
+    if (update.updateType === 'minor' && update.separateMultipleMinor) {
+      const newMinor = String(update.newMinor);
+      update.groupSlug = `minor-${newMinor}-${update.groupSlug}`;
+    }
```

Comment:
Where/how is branch name modified when it's not a group?

Reply:
Yeah.. this confused me and might explain why it feels wrong, but i assumed there would be similar logic for `separateMultipleMajor` but i couldn't see how it was pulling this off

Right now its only working due to the update of the default config for branchTopic now that i think about it again.

looking via the debugger i see that branchName is just the template that the config provides.

what would be the correct way to manipulate this template  to pull off what ive done to the default? as i assume users may adjust this 

some sort of replace from 
`{{{depNameSanitized}}}-{{{newMajor}}}{{#if separateMinorPatch}}{{#if isPatch}}.{{{newMinor}}}{{/if}}{{/if}}.x{{#if isLockfileUpdate}}-lockfile{{/if}}` 
to 
`{{{depNameSanitized}}}-{{{newMajor}}}.{{{newMinor}}}.x{{#if isLockfileUpdate}}-lockfile{{/if}}`

Feels wrong as its user adjustable.

So i assume instead we should append something to the `additionalBranchPrefix` but that would end up with a branch of like `renovate/minor-1.25-amazon-eks-1.x`

or am i just over thinking this?

Reply:
I think we don't need to change anything in this PR, thanks